config/app.php

```diff
@@ -88,6 +88,21 @@
 
     /*
     |--------------------------------------------------------------------------
+    | Hero Text Enabled
+    |--------------------------------------------------------------------------
+    |
+    | This enables the hero image to have excerpt text and a view more link.
+    | You can also specify which controllers you want this to work on by
+    | adding to the array. By default only the homepage controller
+    | is allowed to have this functionality.
+    |
+    */
+    'hero_text_enabled' => true,
+    'hero_text_controllers' => ['HomepageController'],
+    'hero_text_more' => 'View More',
```

Comment:
Let's sentence case this to conform to university style. "View more"

Reply:
I'll get that change into the next feature branch I'm working on!